service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -19,13 +22,13 @@
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
 import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
 public class EmailNotificationHelper {
-    private static final HearingBookingService hearingBookingService = new HearingBookingService();
 
-    private EmailNotificationHelper() {
-    }
+    private final HearingBookingService hearingBookingService;
 
-    public static String buildSubjectLine(final CaseData caseData) {
+    public String buildSubjectLine(final CaseData caseData) {
```

Comment:
Maybe not in this PR so could be a new tech debt ticket but do you think this method and then subsequently the one below should be refactored to only take the required params, that is a lot of data being passed around in case data that is irrelevant to the method

Reply:
I agree and should be handled in a separate ticket.